### DISCUSSION TITLE
How are WSL2 and Windows 11 Sandbox related?

### DISCUSSION THREAD

**Discussion by mhechthz** [microsoft/WSL#9088] [General]
In issue https://github.com/microsoft/WSL/issues/8620 I documented, that **WSL2** and **Windows 11 Sandbox** have identical and synchronized connectivity issues. It seems that the problem is originated by a Windows module/system below both programs.

Has anyone any idea about this?

**Comment by NotTheDr01ds** [comment]
@mhechthz This would likely indicate that the root problem is somewhere in the Hyper-V component.  Both Sandbox and the WSL2 Virtual Machine Platform are built using Hyper-V components.  They don't, however, *require* the Hyper-V "feature" itself to be turned out, which allows them to run on Windows Home SKUs).

**Reply by mhechthz** [reply]
Is there any "average user" compatible description of Hyper-V, i.e. is there a chance to adopt settings in Hyper-V?

**Reply by NotTheDr01ds** [reply]
@mhechthz That, of course, is a good follow-up question ;-).

Unfortunately, there aren't many settings in Hyper-V that we can change ourselves as users of WSL, at least (and I presume Sandbox as well).  For that reason, I personally find networking issues to often be difficult for me to help others troubleshoot.

But I'll noodle on it and see if I can think of any other ideas for #8620 since you've ruled out VPN issues.

**Reply by NotTheDr01ds** [reply]
Also note that (from reading other issues/comments here on Github) the WSL team is separate from the Hyper-V team, so if the root-cause is on the Hyper-V end, sometimes it's out of the WSL team's hands.  They do, of course, have a relationship with that team, but they clearly have a dependency there over which they don't have "full control".

**Comment by anushhkumar** [comment]
Unfortunately, there aren't many settings in Hyper-V that we can change ourselves as users of [WSL](https://forums.autodesk.com/t5/autocad-forum/unhandled-access-violation-reading-0x0003-exception-at-65180867h/td-p/1408194), at least (and I presume Sandbox as well). For that reason, I personally find networking issues to often be difficult for me to help others troubleshoot.
